canal/handler.go

```diff
@@ -17,18 +18,22 @@ type EventHandler interface {
 	OnGTID(header *replication.EventHeader, gtid mysql.GTIDSet) error
 	// OnPosSynced Use your own way to sync position. When force is true, sync position immediately.
 	OnPosSynced(header *replication.EventHeader, pos mysql.Position, set mysql.GTIDSet, force bool) error
+	// OnQueryEvent is query event include (create user, drop user, create index, etc.)
+	// Note: the OnQueryEvent has lower priority than OnDDL even
+	OnQueryEvent(header *replication.EventHeader, stmt ast.StmtNode, pos mysql.Position, e *replication.QueryEvent) (bool, bool, error)
```

Comment:
Please use named return in this interface, or explain what's the meaning of two `bool`s in comments. So users of this library will understand when they implement this interface.